pegasus/sites.v3/hourofcode.com/public/index.haml

```diff
@@ -95,8 +95,7 @@ social:
     .wrapper
       %h2=hoc_s(:hoc_live_join_us)
       %h3=hoc_s(:hoc_homepage_join_the_movement_description)
-      -if ["pre-hoc", "soon-hoc"].include?(hoc_mode)
```

Comment:
This logic is now handled here: https://github.com/code-dot-org/code-dot-org/blob/e76492737ac278d472b1d335370dcc53be807a33/pegasus/sites.v3/hourofcode.com/views/home_signup.haml#L6

Reply:
I'm not clear how this is separate yet. Checking...
`if ["pre-hoc", "soon-hoc"].include?(hoc_mode)` and 
`if ["actual-hoc", "post-hoc", false].include?(hoc_mode)`
seem like opposite checks?

Reply:
I'm not sure I understand the question, can you clarify what you mean?

Reply:
@ebeastlake is correct that they are opposite checks, and @kelbyhawn is correct that the logic is handled elsewhere, specifically by the `else` case :)

https://github.com/code-dot-org/code-dot-org/blob/e76492737ac278d472b1d335370dcc53be807a33/pegasus/sites.v3/hourofcode.com/views/home_signup.haml#L6-L8